### PR TITLE
refactor: revert changes in x/slashing proto

### PR DESCRIPTION
**Pull request by 0Tech**
## Description

closes: #616

## Checklist:

- [x] I followed the [contributing guidelines](https://github.com/line/lbm-sdk/blob/main/CONTRIBUTING.md) and [code of conduct](https://github.com/line/lbm-sdk/blob/main/CODE_OF_CONDUCT.md).
- [x] I have added a relevant changelog to `CHANGELOG.md`
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation accordingly.
- [x] I have updated API documentation `client/docs/swagger-ui/swagger.yaml`

### CODE REVIEW

File: x/slashing/handler_test.go [hunk 1]
```diff
@@ -165,7 +165,7 @@ func TestHandleAbsentValidator(t *testing.T) {
 	// will exist since the validator has been bonded
 	info, found := app.SlashingKeeper.GetValidatorSigningInfo(ctx, sdk.ConsAddress(val.Address()))
 	require.True(t, found)
-	require.Equal(t, int64(0), info.VoterSetCounter)
+	require.Equal(t, int64(0), info.IndexOffset)
```
Comment (dudong2) [inline x/slashing/handler_test.go:168]: The cosmos-sdk also has a StartHeight check. Please add it in this file.

File: x/slashing/keeper/keeper_test.go [hunk 2]
```diff
@@ -240,6 +241,12 @@ func TestValidatorDippingInAndOut(t *testing.T) {
 	// 100 times + (kicked) + 501 times + 398 times = 999 times
 	tstaking.CheckValidator(valAddr, stakingtypes.Bonded, false)
 
+	// check all the signing information
+	signInfo, found := app.SlashingKeeper.GetValidatorSigningInfo(ctx, consAddr)
+	require.True(t, found)
+	require.Equal(t, int64(501), signInfo.MissedBlocksCounter)
+	require.Equal(t, int64(999), signInfo.IndexOffset)
```
Comment (dudong2) [inline x/slashing/keeper/keeper_test.go:248]: The cosmos-sdk also has a StartHeight check. Please add it.

File: x/slashing/keeper/keeper_test.go [hunk 3]
```diff
@@ -250,10 +257,10 @@ func TestValidatorDippingInAndOut(t *testing.T) {
 	tstaking.CheckValidator(valAddr, stakingtypes.Unbonding, true)
 
 	// check all the signing information
-	signInfo, found := app.SlashingKeeper.GetValidatorSigningInfo(ctx, consAddr)
+	signInfo, found = app.SlashingKeeper.GetValidatorSigningInfo(ctx, consAddr)
 	require.True(t, found)
 	require.Equal(t, int64(0), signInfo.MissedBlocksCounter)
-	require.Equal(t, int64(1000), signInfo.VoterSetCounter)
+	require.Equal(t, int64(0), signInfo.IndexOffset)
```
Comment (dudong2) [inline x/slashing/keeper/keeper_test.go:263]: Ditto.

**Review by dudong2 — APPROVED** [review]

File: simapp/export.go [hunk 4]
```diff
@@ -180,7 +180,7 @@ func (app *SimApp) prepForZeroHeightGenesis(ctx sdk.Context, jailAllowedAddrs []
 	app.SlashingKeeper.IterateValidatorSigningInfos(
 		ctx,
 		func(addr sdk.ConsAddress, info slashingtypes.ValidatorSigningInfo) (stop bool) {
-			info.VoterSetCounter = 0
+			info.StartHeight = 0
```
Comment (zemyblue) [inline simapp/export.go:183]: Please fix the above comment. 
```
// reset voter set counter on signing infos
```
-->
```
// reset start height on signing infos
```

File: x/slashing/handler_test.go [hunk 5]
```diff
@@ -165,7 +165,7 @@ func TestHandleAbsentValidator(t *testing.T) {
 	// will exist since the validator has been bonded
```
Comment (zemyblue) [inline x/slashing/handler_test.go:165]: Please check this comment also.

File: x/slashing/handler_test.go [hunk 6]
```diff
@@ -258,7 +258,7 @@ func TestHandleAbsentValidator(t *testing.T) {
 	// Validator voter set counter should not have been changed
 	info, found = app.SlashingKeeper.GetValidatorSigningInfo(ctx, sdk.ConsAddress(val.Address()))
 	require.True(t, found)
-	require.Equal(t, int64(1502), info.VoterSetCounter)
+	require.Equal(t, int64(1), info.IndexOffset)
```
Comment (zemyblue) [inline x/slashing/handler_test.go:261]: In this case, how about checking `info.StartHeight` also?

File: x/slashing/handler_test.go (hunk 5 above)
Comment (0Tech) [reply x/slashing/handler_test.go:165]: Could you please elaborate on your comment?

Comment (zemyblue) [reply x/slashing/handler_test.go:165]: I pointed wrong line. Sorry.

**Review by zemyblue — APPROVED** [review]

**Pull request merged**